SYCL/ESIMD/api/functional/type_traits.hpp

```diff
@@ -27,6 +29,13 @@ template <typename T>
 inline constexpr bool is_sycl_floating_point_v{
     is_sycl_floating_point<T>::value};
 
+template <typename T>
+using is_sycl_signed =
+    std::bool_constant<std::is_signed_v<T> || std::is_same_v<T, sycl::half>>;
```

Comment:
```suggestion
    std::bool_constant<std::is_signed_v<T> || is_sycl_floating_point_v<T>>;
```
Nit.
Just as a suggestion for DRY

Reply:
I've applied this comment [here](https://github.com/intel/llvm-test-suite/pull/789/commits/41df5e531f17caed6253334af281f97f021058d3#diff-7b267ff200ef7dbfe5e2b1fe912dcbc7a8622f4d14a995f2f683282cff5e1b49R34)